opm/input/eclipse/EclipseState/Tables/TableManager.hpp

```diff
@@ -417,6 +420,7 @@ namespace Opm {
         double m_rtemp {288.7056}; // 60 Fahrenheit in Kelvin
         double m_salinity {0.0};
         int m_actco2s {3};
+        bool m_diff_mole_fraction {true};
```

Comment:
Does this mean we default to enabling this feature?  If so, I think we should discuss this a bit&ndash;at least if there's computational cost associated to having the feature on.

Reply:
You still need to enable diffusion. This just mean that if you dont set the DIFFAWAT of DIFFAGAS keyword we use the mole fraction formulation.